service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsAboutToSubmitControllerTest.java

```diff
@@ -49,6 +50,9 @@
     @MockBean
     private RequestData requestData;
 
+    @MockBean
+    private PaymentService paymentService;
```

Comment:
looks like this is not used so should be fine to removed

Reply:
The issue was related to auto-wiring. If I don't add the MockBean here and the test was failing, hence added it. 
Is there any other better way of doing it?

Reply:
discussed

Reply:
yeah , that was due to earlier package structure ...  after moving class to another package this change is not required.
Thanks Toireasa